lightning/src/chain/channelmonitor.rs

```diff
@@ -2463,7 +2507,7 @@ impl<Signer: WriteableEcdsaChannelSigner> ChannelMonitorImpl<Signer> {
 		for update in updates.updates.iter() {
 			match update {
 				ChannelMonitorUpdateStep::LatestHolderCommitmentTXInfo { commitment_tx, htlc_outputs, claimed_htlcs, nondust_htlc_sources } => {
-					log_trace!(logger, "Updating ChannelMonitor with latest holder commitment transaction info");
+					log_trace!(WithChannelMonitorImpl::from(logger, self), "Updating ChannelMonitor with latest holder commitment transaction info");
```

Comment:
i dont think i have noticed any instances where channel id's might change. 

> to just use a wrapper that makes a copy when initializing instead of wrapping each time we log.
does that mean we will prefer the copy version of these wrappers? should i make the change to use those instead everywhere?